enterprise/frontend/src/embedding-sdk/hooks/private/setup-sdk-auth.ts

```diff
@@ -1,50 +0,0 @@
-import { P, match } from "ts-pattern";
```

Comment:
I removed this file and inlined the logic [here](https://github.com/metabase/metabase/pull/49214/files#diff-7242fd1ce143f9ed3c058d8bd5e4820c9e310fb2f1d79b5b6854c4528b1b0d53R18-R38).
I wanted to make it clearer what calls could have thrown an error, having to look in another file was hiding it a bit